src/components/custom/other-platforms.js

```diff
@@ -344,7 +344,7 @@ export const NavMarket = ({ onClick, is_ppc }) => (
             }
             title={<Localize translate_text="Forex" />}
             onClick={onClick}
-            to="/markets#forex"
+            to="/markets/forex"
```

Comment:
@njazuli-deriv The links should all end with a trailing slash since our sitemap links are also generated that way 🙏

Reply:
added trailing slash in latest commit.